administrator/components/com_users/tmpl/notes/default.php

```diff
@@ -97,7 +97,7 @@
                                 </div>
                             </th>
                             <td class="d-none d-md-table-cell">
-                                <?php echo $this->escape($item->user_name); ?>
+                                <?php echo !empty($item->user_name) ? $this->escape($item->user_name) : Text::_('JNONE'); ?>
```

Comment:
```suggestion
                                <?php echo !empty($item->user_name) ? $this->escape($item->user_name) : '[ ' . Text::_('JNONE') . ' ]'; ?>
```
Someone could name a user "None". To distinguish that, the list view should show "[ None ]" like PR #45232 does it e.g. here: https://github.com/joomla/joomla-cms/pull/45232/files#diff-bd22b34aa4cc58086d9e4756bbdb29628f0950b4723c592ef5ebc6f57bdf92dcR329

Reply:
The other PR has it as standard markup vs PHP concatenation.

Reply:
Yes that's a difference, but it does the same, wrap it into squared brackets with spaces.